app/car.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+from dataclasses import dataclass
+
+
+@dataclass
+class Car:
+    brand: str
+    fuel_consumption: int | float
+
+    @classmethod
+    def get_car_info(cls, car: dict) -> Car:
```

Comment:
`get_car_info` - incorrect name of this function, and why do you crate this method, you can create instance without this method